.env.example

```diff
@@ -12,7 +12,7 @@ ONBOARDING_CLIENT_ID=...
 ONBOARDING_CLIENT_SECRET=...
 ONBOARDING_ENVIRONMENT_ID=us_env_...
 ONBOARDING_SPACE_CONFIG_ID=us_sc_...
-WORKBOOK_UPLOAD_WORKBOOK_NAME="HCM Workbook"
+ONBOARDING_WORKBOOK_NAME=MyWorkbook123
```

Comment:
@mhpjay422 Before merging add this new var to heroku, and after it's deployed remove the old one

Reply:
added new var to heroku

Reply:
deployed and removed the old one